src/searchengine/nova3/engines/demonoid.py

```diff
@@ -129,6 +130,7 @@ def search(self, what, cat='all'):
         connection.request("GET", query)
```

Comment:
What happens if the domain is blocked or no longer exists? What happens if the server is down and there is no answer? I think in both cases there is an exception...